decidim-core/app/types/decidim/core/base_input_sort.rb

```diff
@@ -6,8 +6,9 @@ class BaseInputSort < GraphQL::Schema::InputObject
       # Overwrite the prepare method to allow 2 possible values only
       def prepare
         arguments.each do |key, value|
-          next if key == "locale"
-          raise GraphQL::ExecutionError, "Invalid order value for #{key}, only ASC or DESC are valids" unless valid_order?(value)
+          next if key.to_sym == :locale
```

Comment:
For security, we should avoid turning something into a symbol if it might come from the end user. Ruby symbols don't get garbage-collected, so it's a possible way to hit the server and make it run out of memory.

```suggestion
          next if key.to_s == "locale"
```